docs/user-permissions.md

```diff
@@ -0,0 +1,60 @@
+# User Permissions
+
+There are a number of permissions that can be applied to a user in Content
+Publisher. In integration, staging and production environments these are
+managed through [signon][]. In a local environment permission changes can be
+made through the Rails console, for example:
+
+```
+irb(main):001:0> user = User.first
+irb(main):002:0> user.permissions << User::ACCESS_LIMIT_OVERRIDE_PERMISSION
+irb(main):003:0> user.save
+```
+
+## Permissions and their affects
+
+All permissions used by Content Publisher are set as constants on the
+[User model][].
+
+### `access_limit_override`
+
+The access limit feature in Content Publisher allows a publisher to forbid
+users outside of their organisation from accessing a document prior to its
+publishing. A user with the `access_limit_override` permission will be
+granted access to all documents no matter what organisation they belong to.
+
+### `debug`
+
+The `debug` permission is intended for Content Publisher developers as a means
+to view diagnostic information within the application. This is currently
+limited to being able to view the edits made in each revision of a document.
+
+### `managing_editor`
+
+The `managing_editor` permission acts as a role that is given to users who have
+the are the managing editors for their department. Having this permission
+enable users to manage the withdrawing of content and to decide whether content
```

Comment:
```suggestion
enables users to manage the withdrawing of content, and to decide whether content
```

Reply:
Thanks - I didn't go with the comma suggestion as that didn't flow right for me so tweaked the wording slightly.